### ISSUE TITLE
Do queued emails get removed on cart recovery?

### ISSUE THREAD

**Issue by sunscreem** [mediabeastnz/craft-commerce-abandoned-cart#17]
Hey Myles. Lets say the first and second emails have been queued.

The first email is sent, and the user clicks through and restores their cart.

What happens to the queued second email job?

**Comment by mediabeastnz** [comment]
There's a check to not actually send out an email if the order is already recovered. However I do need to change this so it removes it from the queue entirely.

**Comment by sunscreem** [comment]
Got it - thanks myles. So the job remains queued, but won't actually email when it gets fired?

**Comment by mediabeastnz** [comment]
Exactly. On my to do list though to completely remove it form the queue.

**Comment by sunscreem** [comment]
Hello Sir,

Just trying to finish up my understanding of how the plug in handles restoring carts and not sending the second email. I'm sorry for the endless stream of questions. 

Looks at the code I think the cart is marked as _recovered_ when an order is placed?

So would this be right:

1. Abandoned cart found, 1st email sent.
2. User clicks the link and their cart is _restored_.
3. They start the checkout process again, but don't complete it.
4. The 2nd email is still still queued and sent? (Hence this open issue)
5. The 1st email won't be triggered to be queued again because its already been sent once?

Rob
